homeassistant/helpers/aiohttp_client.py

```diff
@@ -25,10 +25,12 @@ def async_get_clientsession(hass, verify_ssl=True):
         key = DATA_CLIENTSESSION_NOTVERIFY
 
     if key not in hass.data:
+        config = hass.config.as_dict()
         connector = _async_get_connector(hass, verify_ssl)
         clientsession = aiohttp.ClientSession(
             loop=hass.loop,
-            connector=connector
+            connector=connector,
+            headers={'User-Agent': 'Home Assistant' + config['version']}
```

Comment:
- import `__version__` from `homeassistant.const`
- Use string formatting
- Also include Python and aiohttp version in user agent string

```python
    SERVER_SOFTWARE = 'HomeAssistant/{0} aiohttp/{1} Python/{2[0]}.{2[1]}'.format(
        __version__, aiohttp.__version__, sys.version_info)
```